vassal-app/src/main/java/VASSAL/counters/Labeler.java

```diff
@@ -262,56 +276,11 @@ public String getLocalizedName() {
     }
   }
 
-
-// FIXME: This doesn't belong here. Should be in ImageUtils instead?
-  public static void drawLabel(Graphics g, String text, int x, int y, int hAlign, int vAlign, Color fgColor, Color bgColor) {
-    drawLabel(g, text, x, y, new Font("Dialog", Font.PLAIN, 10), hAlign, vAlign, fgColor, bgColor, null);
-  }
-
-// FIXME: This doesn't belong here. Should be in ImageUtils instead?
-  public static void drawLabel(Graphics g, String text, int x, int y, Font f, int hAlign, int vAlign, Color fgColor, Color bgColor, Color borderColor) {
-    g.setFont(f);
-    final int width = g.getFontMetrics().stringWidth(text + "  ");
-    final int height = g.getFontMetrics().getHeight();
```

Comment:
Shouldn't these be deprecated and defer to LabelUtils.drawLabel()? I use these all over the place in my custom code as a quick & nasty to paste odd bits of info on top of counters.

Reply:
Ah Sorry, didn't scroll down far enough. Looks good.